pvlib/ivtools.py

```diff
@@ -0,0 +1,262 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 29 10:34:10 2019
+
+@author: cwhanse
+"""
+
+import numpy as np
+
+
+def fit_cec_with_sam(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
+                     gamma_pmp, cells_in_series, temp_ref=25):
+    '''
+    Estimates parameters for the CEC single diode model using the SAM SDK.
```

Comment:
Perhaps reference "An Improved Coefficient Calculator for the CEC 6 Parameter Photovoltaic Module Model" or the SAM SDK documentation here?

Reply:
Added Dobos reference. The SAM SDK function 6parsolve isn't documented anywhere that I could find.